addons/AudioPlaylist/documentation.md

```diff
@@ -1 +1,359 @@
-To be done later.
+## Description
+The Audio Playlist addon allows users to create a list of sounds and play them. This addon works on all common browsers.
+<div style="border:1px solid Tomato; padding:5px; margin-bottom:21px;">
+Note: It's recommended to use files with <a href="http://en.wikipedia.org/wiki/Bit_rate">bitrate</a> 64 kb/s or higher. Files with lower quality could have difficulties with reproduction.
+</div>
+
+## Properties
+
+<table border='1'>
+    <tr>
+        <th>Property name</th>
+        <th>Description</th>
+    </tr>
+    <tr>
+    <td>Items</td>
+        <td>
+            This property serves for storing audio items.
+                <table border='1'>
+                    <tr>
+                        <th>Property name</th>
+                        <th>Description</th>
+                    </tr>
+                    <tr>
+                        <td>Name</td>
+                        <td>Audio name displays in the list.</td>
+                    </tr>
+                    <tr>
+                        <td>Mp3</td>
+                        <td>This property serves for uploading an audio file in mp3 format into the module.</td>
+                    </tr>
+                    <tr>
+                        <td>Ogg</td>
+                        <td>This property serves for uploading an audio file in ogg format into the module. It's needed for all common browsers.</td>
```

Comment:
Czy ten fragment "It's needed for all common browsers." jest potrzebny ? 
Z dokumentacji dla Audio ten fragment usunąłeś.